cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -399,19 +400,26 @@ def _validate_curie(self, term_id: str, column_name: str, curie_constraints: dic
 
         :rtype None
         """
+        # Check if term_id is forbidden by schema definition. Sometimes, these are also invalid ontology
+        # terms, but it's preferred to report these as "not allowed" terms rather than "invalid ontology terms"
+        if "forbidden" in curie_constraints:
```

Comment:
Is there different if login here or is this just a reorganizing of logic?

Reply:
no, it's just a reorganizing of logic so the forbidden terms check can occur before validating that they're valid ontology terms. but they're still required to be valid ontology terms before proceeding into the forbidden ancestors check.